lib/cadet/course/materialUpload.ex

```diff
@@ -1,13 +1,15 @@
-defmodule Cadet.Course.Upload do
+defmodule Cadet.Course.MaterialUpload do
   @moduledoc """
-  Represents an uploaded file
+  Represents an uploaded file for Material
   """
   use Arc.Definition
   use Arc.Ecto.Definition
 
-  @extension_whitelist ~w(.doc .docx .jpg .pdf .png .ppt .pptx .txt .wav .xls .xlsx)
+  @extension_whitelist ~w(.doc .docx .jpg .pdf .png .ppt .pptx .txt .xls .xlsx)
   @versions [:original]
 
+  def bucket, do: "cadet-materials"
```

Comment:
this is way to brittle man! we should throw this into config to decouple the app from the bucket name

Reply:
Refactored.